### PR TITLE
[_]: Refactor/cleaner scripts to TS

### PR DESCRIPTION
**Pull request by fabioespinosa**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "inxt-audit": "bin/inxt-audit-monitor.js"
   },
   "scripts": {
+    "build:commands": "tsc",
```
Comment (sg-gs) [inline package.json]: This command already exists

File: package.json [hunk 2]
```diff
@@ -97,6 +103,7 @@
     "lodash": "^4.17.20",
     "merge": "^1.2.1",
     "mime-db": "^1.40.0",
+    "mongoose": "^6.2.6",
```
Comment (sg-gs) [inline package.json]: remove mongoose from deps or if it is required, move it to dev dependencies

File: package.json [hunk 3]
```diff
@@ -62,12 +66,14 @@
...
     "memdown": "^4.1.0",
     "mocha": "^6.2.0",
+    "mysql": "^2.18.1",
     "node-mocks-http": "^1.7.6",
     "nodemon": "^2.0.15",
     "noisegen": "^1.0.0",
     "proxyquire": "^1.7.10",
     "redis-mock": "^0.16.0",
     "rimraf": "^2.6.3",
+    "typescript": "^4.6.2",
```
Comment (sg-gs) [inline package.json]: typescript is already installed as dependency, so it could be removed from dev dependencies

**Review by sg-gs — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (fabioespinosa) [reply package.json]: fixed

File: package.json (hunk 2 above)
Comment (fabioespinosa) [reply package.json]: fixed

Comment (fabioespinosa) [reply package.json]: (removed)

File: package.json (hunk 3 above)
Comment (fabioespinosa) [reply package.json]: fixed

**Review by sg-gs — APPROVED** [review]

**Pull request merged**